extensionsGallery-insider.json

```diff
@@ -220,7 +220,7 @@
 							},
 							{
 								"assetType": "Microsoft.VisualStudio.Services.Content.Details",
-								"source": "https://foundrylabssqlsearch.blob.core.windows.net/sqlops-sql-search-insiders/metadata/README.md"
+							 "source": "https://foundrylabssqlsearch.blob.core.windows.net/sqlops-sql-search-insiders/metadata/README.md"
```

Comment:
spacing is wrong

Reply:
@llali I copy and pasted this text from the `stable` gallery file.  I'd rather not change the `stable` file now since then I'd have to redeploy that file as well.  I think we should fix the spacing in the `stable` file next time it's edited and then pickup the reformatting next time we refresh the `insiders` file.